### PR TITLE
updated lang for opensource

### PR DESCRIPTION
**Pull request by piamancini**
made some tweaks in open source open collective.

### CODE REVIEW

File: frontend/src/ui/strings.json [hunk 1]
```diff
@@ -259,5 +259,11 @@
     "membersWallText": "Proud backers and sponsors support this collective and allow us to keep going towards our mission. We would not be here if it weren’t for these amazing people. Thank you so much.",
     "membersWallTitle": "This is possible thanks to you.",
     "menuMembersWall": "Supporters"
+  },
+  
+  "opensource":{
+    "menuMembersWall": "Supporters"
+    "becomeMemberTitle": "Why support this Collective?"
```
Comment (asood123) [inline frontend/src/ui/strings.json]: need commas on previous lines (that's why the red boxes appearing)

File: frontend/src/ui/strings.json [hunk 2]
```diff
@@ -259,5 +259,11 @@
...
+    "joinAndFulfil": "Support this collective and help bring sustainable funding to Open Source",
```
Comment (asood123) [inline frontend/src/ui/strings.json]: no comma at the end

**Comment by piamancini** [comment]
thank you, fixed commas.

File: frontend/src/ui/strings.json [hunk 3]
```diff
@@ -259,5 +259,11 @@
     "membersWallText": "Proud backers and sponsors support this collective and allow us to keep going towards our mission. We would not be here if it weren’t for these amazing people. Thank you so much.",
     "membersWallTitle": "This is possible thanks to you.",
     "menuMembersWall": "Supporters"
+  },
+  
+  "opensource":{
+    "menuMembersWall": "Supporters",
+    "becomeMemberTitle": "Why support this Collective?",
```
Comment (asood123) [inline frontend/src/ui/strings.json]: fyi, "Collective" is capitalized here but not in the next one. Is that intentional?

**Comment by asood123** [comment]
Besides the capital issue, rest looks good. Feel free to merge.

**Pull request merged**